CMakeLists.txt

```diff
@@ -228,6 +228,7 @@ if(OpenCL_ENABLE)
             ENV AMDAPPSDKROOT
             ENV ATISTREAMSDKROOT
             ENV "PROGRAMFILES(X86)"
+	    /usr/include
```

Comment:
please fix the indention and add only `/usr` because `include` is selected by the suffix

Reply:
Updated.